code-env/python/desc.json

```
@@ -1,5 +1,5 @@
 {
-  "acceptedPythonInterpreters": ["PYTHON27", "PYTHON35", "PYTHON36"],
+  "acceptedPythonInterpreters": ["PYTHON35", "PYTHON36"],
```

Py35 is not supported anymore
```suggestion
  "acceptedPythonInterpreters": ["PYTHON36"],
```

---

Done in c43aac3